content/realtime/messages.textile

```diff
@@ -160,6 +170,11 @@ bc[swift]. channel.subscribe { message in
     print("message data: \(message.data)")
 }
 
+bc[flutter]. final channelMessageSubscription = channel.subscribe().listen((ably.Message message){
+ print('message received for event ${message.name}');
```

Comment:
Inconsistent indentation.

Reply:
addressed in e288898cb0f972b74350bd69c2da054346fbb4f0

Reply:
@tiholic Can you revisit your fix, please. You're missing a space before the opening curly brace [here](https://github.com/ably/docs/commit/e288898cb0f972b74350bd69c2da054346fbb4f0#diff-ab0455ad7012decae9dc129170e4c90888957d9d6fc2f14ca30fa6ba601b3586R175).

Reply:
Thanks for pointing this out @QuintinWillison. There were quite a number of usages where I missed that space.
Addressed 19a969ae337a59f8d8c3c8d8d4bcde6858dbd91c

Reply:
Thanks. I'm going to resolve this conversation, on behalf of Matt.

Reply:
@Quintin
In that case we're good to resolve this conversation, I guess _( unless the syntax isn't clear enough! )_